### PR TITLE
It1 update report model

### LINKED ISSUE #63: Create mocked data for REPORT return for iteration 1 updates to salary, monthly debt, pmi, tax, insurance, and mortgage term

### LINKED ISSUE #81: Update Report model functions/methods
This model needs updates to:

- [ ] Downpayment calculations (account for salary, debt, and rent/monthly_living)
- [ ] Zip to city-state
- [ ] Zip to avg home price
- [ ] Any relevant function changes

### LINKED ISSUE #82: Update Report Controller for changes made for mocked data

### PR DESCRIPTION
**Pull request by priyapower**
# This PR:
---

## Changes
- Updates Report Controller for formatting in JSON response per iteration 1 designs
  - Returns some sort of order
  - Updates downpayment plan to include details that are customized to user (takes into account salary and monthly debt now)
- Also updates Report functions:
  - NEW: number_of_years
  - UPDATES: downpayment_goal_monthly_savings
  - NEW: city_state
  - NEW: home_price_by_zip 
- **NEEDS TEST UPDATES (see Issue #80)**
- [X] Adds code for functionality
- [ ] Adds a test
- [ ] Updates Documentation/ReadMe
- [ ] Adds dependencies
- [ ] Refactors code
- [ ] Fixes a bug

## Checklist
**Please check off what you can for this PR:**
- [ ] This code passes all tests on local machine
- [ ] This code is DRY
- [X] This code follows OOP
- [X] This code, if updates to functionality were made, updates all corresponding files

**The "I have" section is a double check for PR specific information:**
- [X] I have tagged reviewers for this PR
- [X] I have checked what branch I am merging into (current expectation is `it1_development`)
- [X] I have added this PR to the project
- [x] I have added this PR to the correct milestone
- [ ] I have updated documenation if changes were made that affect setup/running

## Linked tickest / issues 
Closes #63 
Closes #81 
Closes #82

### CODE REVIEW

File: api/controller/EducationController.py [hunk 1]
```diff
@@ -12,15 +12,17 @@ def get_education(self, id):
...
+                    "02_id": str(education.id),
+                    "03_attributes": {
+                        "A_order": education.order,
+                        "B_classification": education.classification,
+                        "C_question": education.question,
+                        "D_description": education.description,
+                        "E_information": education.information,
+                        "F_note": education.note,
+                        "G_source": education.source,
+                        "H_symbol": education.symbol
```
Comment (BlakeDonn) [inline api/controller/EducationController.py:25]: Thanks for alphabetizing these, definitely nice to not have to sort on our end :)

File: api/models/report.py [hunk 2]
```diff
@@ -4,14 +4,15 @@
...
     zipcode = db.IntField(required=True)
     credit_score = db.IntField(required=True)
+    salary = db.IntField(required=True)
     monthly_debt = db.IntField(required=True)
     downpayment_savings = db.IntField(required=True)
+    mortgage_term = db.IntField(default=0)
     downpayment_percentage = db.IntField(required=True)
-    rent = db.IntField(default=0)
     goal_principal = db.IntField(default=0)
+    rent = db.IntField(default=0)
```
Comment (BlakeDonn) [inline api/models/report.py:15]: Cool, glad to see we have some defaults so we can let the use know they aren't required on the frontend!

**Review by BlakeDonn — APPROVED** [review]
Looks great! These will definitely help us out on the frontend.

File: api/controller/EducationController.py [hunk 3]
```diff
@@ -46,15 +48,17 @@ def convert_snake(string):
...
```
Comment (Arique1104) [inline api/controller/EducationController.py:61]: I now understand where H_symbol hit this scene.  We need a mascot for H_symbol!

File: api/models/education.py [hunk 4]
```diff
@@ -7,3 +7,5 @@ class Education(db.Document):
     information = db.StringField()
     note = db.StringField()
     source = db.StringField(required=True)
+    order = db.IntField(required=True)
+    symbol = db.StringField(default="")
```
Comment (Arique1104) [inline api/models/education.py:11]: Lovely catch @priyapower!

File: database/education.py [hunk 5]
```diff
@@ -1,72 +1,87 @@
...
+credit = {
+    "classification": "Credit Score",
+    "question": "What is your current credit score?",
+    "description": " A high score will give you access to lower interest rates and more lender choices. If you have a low score, you may have trouble getting a loan.",
+    "information": "Your credit score plays a big role in the interest rate you’ll get for your loan.",
+    "note": "Your credit score is a numerical rating that ranges from 300 – 850 and tells lenders how responsible you are when you borrow money. ",
+    "source": "https://www.rocketmortgage.com",
+    "order": 2
+}
+salary = {
```
Comment (Arique1104) [inline database/education.py:19]: Thank the globs that we finally got this content updated!!!  And thank you @priyapower for securing it!

File: database/education.py [hunk 6]
```diff
@@ -1,72 +1,87 @@
...
+    "source": "https://www.rocketmortgage.com",
+    "order": 3,
+    "symbol": "$"
 }
 debt = {
     "classification": "Monthly Debt",
     "question": "What would you say your monthly debt is?",
-    "description": "What to include: Minimum credit card payment, Auto, student, or personal loan payments,Alimony or child support, Do not include: Current rent or mortgage Credit card balances Monthly utilities, groceries, and other costs of living",
-    "information":"This is NOT your DTI aka Debt-To-Income Ratio, Your DTI is equal to your total fixed, recurring monthly debts divided by your total monthly gross household income.",
+    "description": "What to include: minimum credit card payment, auto, student, or personal loan payments, alimony or child support. Do not include: current rent or monthly utilities, groceries, and other costs of living.",
```
Comment (Arique1104) [inline database/education.py:32]: Great catch!

**Review by Arique1104 — APPROVED** [review]
Looks great!

File: api/models/report.py (hunk 2 above)
Comment (priyapower) [reply api/models/report.py:15]: Absolutely!

File: api/models/education.py (hunk 4 above)
Comment (priyapower) [reply api/models/education.py:11]: Thank you!

File: database/education.py (hunk 5 above)
Comment (priyapower) [reply database/education.py:19]: I still believe we can flush these out more. I would like to ask the team to give their input and see if any phrases can be updated

**Pull request merged**